### ISSUE TITLE
Milestone detail page to show both issues and pull requests

### ISSUE THREAD

**Issue by harupong** [takezoe/gitbucket#892]
While GitHub shows both issues and pull requests when you click on any milestones, GitBucket only shows issues.

Is GitBucket capable of showing both issues and pull requests under the same milestone, like what you get when you access https://github.com/takezoe/gitbucket/milestones/Priority1 ?

**Comment by imeszaros** [comment]
I agree, milestone listing counts related PR-s also when computes the milestone progress, but the details list only shows the issues
